src/App.js

```diff
@@ -8,13 +8,17 @@ import {
     Route
   } from "react-router-dom";
 import About from './Pages/About';
+import Login from './Pages/Login';
 
 const App = () => {
     return (
       <div className="body">
         <div className="main">
           <Router>
             <Switch>
+              <Route path="/login">
```

Comment:
You can pass a `component` prop to the Route component, as `Component={<Login/>}`